krish98sai/geo-puzzle
# React native initialized

To test:

1. Clone the repo
2. git checkout react-native
3. Follow the README instructions for "Development Environment Setup for React Native Application" from step 3.
4. Ensure that the application starts and updates to the code also update the app.
